test/raven.utils.js

```diff
@@ -98,6 +98,66 @@ describe('raven.utils', function() {
     });
   });
 
+  describe('#prepareInitialRequest', function() {
```

Comment:
these tests on the prepareInitialRequest utility seem good to me, but would you mind writing a slightly broader test that recreates the original problem outlined in #360?

Reply:
Done.